polyhost/server/control_server.py

```
@@ -0,0 +1,296 @@
+            targets = [(c, self._conn_locks.get(c)) for c in self._subscribed]
+        if not targets:
+            return
+        event = p.make_event(name, payload)
+        dead = []
+        for conn, lock in targets:
+            if lock is None:
+                continue
+            try:
+                with lock:
+                    p.send_message(conn, event)
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)